### ISSUE TITLE
Use https protocol to load prebuild mongodb image

### ISSUE THREAD

**Issue by mikhail-angelov** [winfinit/mongodb-prebuilt#46]
It's better to use https protocol to load mongodb binary
e.g. 
current load url  is:
http://downloads.mongodb.org/linux/mongodb-linux-x86_64-debian71-latest.tgz
but resource is available via 
https://downloads.mongodb.org/linux/mongodb-linux-x86_64-debian71-latest.tgz
as well

it may help to use this library for unit tests on free plans on service CI or cycle CI
currently it hungs with error like this
```
  mongodb-download-MongoDBDownload isDownloadPresent() location missing: false +0ms
(node:111) UnhandledPromiseRejectionWarning: 
Unhandled promise rejection (rejection id: 2): Error: Protocol "http:" not supported. Expected "https:"
Too long with no output (exceeded 10m0s)
```

**Comment by IhostVlad** [comment]
We has solved this problem by following workaround: fix `mongodb-download` package version in package.json to `2.2.3` - after it works very well

```
"mongo-unit": "^1.2.0",
"mongodb-download": "2.2.3",
"mongodb-prebuilt": "^6.3.4",
```

But of course problem still faced and should be fixed by `mongodb-prebuild` maintainers

**Comment by LoneRifle** [comment]
Should be fixed by winfinit/mongodb-download#27
